src/util/keys.ts

```diff
@@ -17,7 +17,9 @@ export const JumpToFadeDurationKey = 'NavigationHandler.JumpToFadeDuration';
 // To get any scene graph node you need ScenePrefix+NodeIdentifier
 export const ScenePrefixKey = 'Scene.';
 
-export const InterestingTagKey = 'GUI.Interesting';
+// @TODO (2024-04-07, emmbr): Remove this key here and from engine - it is no longer used
```

Comment:
Why is it no longer used? Is it replaced by the marked nodes?

Reply:
Yes, this tag was previously used to identify the marked nodes